Test2/Pages/HistoryPage.cs

```diff
@@ -0,0 +1,74 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using SeleniumExtras.WaitHelpers;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Test2.Pages
+{
+    internal class HistoryPage 
+    {
+        private IWebDriver driver;
+        public HistoryPage(IWebDriver webDriver)
+        {
+            driver = webDriver;
+        }
+        public List<IWebElement> LastResult => driver.FindElements(By.XPath("//table/tr[@class='data-td'][1]/td")).ToList();
+        public List<IWebElement> Rows => driver.FindElements(By.XPath("//table/tr[@class='data-td']")).ToList();
```

Comment:
You use this 'Rows' only to get history size, so you can make it shorter and re-use the 'TableValues' property:
```suggestion
        public int HistoryCount => TableValues.Count() - 1;
```